waterlog/src/actions/TankLevelsActions.js

```diff
@@ -20,12 +20,13 @@ import {
     payload: { error }
   });
   
-  export const fetchTankLevels = () => dispatch => {
+  export const fetchTankLevels = () => dispatch => { 
     dispatch(fetchTankLevelsBegin());
-    fetch(`${Globals.API_URL}/api/Tanklevels`)
+    fetch(`${Globals.API_URL}/api/tanklevels`)
       .then(handleErrors)
       .then(res => res.json())
       .then(levels => {
+        console.log('levels', levels)
```

Comment:
Do you wanna test this in Dev or you forgot to remove the console.log('levels', levels)

Reply:
I removed those comments